scripts/run_integration_tests.sh

```diff
@@ -23,7 +23,8 @@ docker run -d --name qgis -v /tmp/.X11-unix:/tmp/.X11-unix \
  -e ONLY_CALC_ID="$ONLY_CALC_ID" \
  -e ONLY_OUTPUT_TYPE="$ONLY_OUTPUT_TYPE" \
  -e GEM_QGIS_TEST=y \
- qgis/qgis:final-3_8_3
+ qgis/qgis:release-3_10
```

Comment:
For now I'd keep final-3_8_3 to avoid segfaults. We can move to release-3_10 as soon as segfaults are fixed